docs/versioned_docs/version-2.50.0-LTS/contributing-guide/setup/docker.md

```diff
@@ -27,19 +27,25 @@ If you are setting up on a Windows machine, we advise you to set up Docker Deskt
 Make sure to run it within the WSL2 terminal.
 :::
 
-1. Clone the repository:
+1. Fork the repository:
+
+   Go to the [ToolJet GitHub repository](https://github.com/ToolJet/Tooljet), click on the 'Fork' button to create a copy of the repository under your own GitHub account.
```

Comment:
Can we use **Fork** instead of 'Fork'?

Reply:
Alright you mean:
click on 'Fork'  ->  click on **Fork**

Reply:
Okay Changing it :)